### PR TITLE
chore: release v0.1.0

### PR DESCRIPTION
**Pull request by b5**
let's do it!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
+# v0.1.0 - 2022-10-28
+
+We’re on the board 🎉! This first release of iroh brings a new implementation of IPFS to the world. 
+
+Key things to highlight about the first release of iroh:
+
+* **Exchange Data with Kubo**. Iroh can interoperate with [kubo](https://github.com/ipfs/kubo) nodes on the IPFS network, pushing & fetching data.
+* **Service Oriented architecture**. Don't want p2p? Turn it off! Iroh will still work without it. This prepares us for
```
Comment (ramfox) [inline CHANGELOG.md:8]: fragment "This prepares us for"

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+
+You'll need to download all of them & put them on your $PATH to work with iroh. Our [install docs](https://iroh.computer/docs/install) have more info.
+
+Please do give iroh a try, we'd love to [hear your feedback](https://github.com/n0-computer/iroh/issues/new). Thanks!
+
+### Benchmarks
+We're runnning benchmarks on a relatively stock AWS box for ease-of-replication. We don't have benchmarks for IPFS network retrieval this round.
+
+### Adding Single Files
+Add a single file of different sizes with `iroh add --no-wrap --offline`:
```
Comment (ramfox) [inline CHANGELOG.md:27]: I think you can just say "Add a single file with `iroh add --no-wrap --offline`"

We get that it's different sizes from the chart

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+
+### Production Readiness
+Is this production ready? **No. We need your help to kick tires & find bugs!**.
+
+The v0.1.0 moniker should testify to just how new this software is. Please don't deploy it anywhere mission critical until we've had at least a month or two to address bugs & write tests.
+
+### Things you should know about this release:
+There are two problems we couldn't properly address
+* _There is no delete command._ Data added to iroh will stay there until a future release where we add content removal.  See [issue #432](https://github.com/n0-computer/iroh/issues/432) for details. The workaround for now is to blow away the storage database.
+* _We have one particularly nasty known bug that can cause iroh to not persist data upon add_ we plan to address ASAP in a patch release. See [issue #423](https://github.com/n0-computer/iroh/issues/423).
```
Comment (ramfox) [inline CHANGELOG.md:56]: Do we know that it occurs in nested directories? Maybe we can be more specific? "that can cause iroh to not persist data for nested directories upon add"

**Review by ramfox — CHANGES_REQUESTED** [review]

**Review by ramfox — APPROVED** [review]
v 0️⃣ . 1️⃣ . 0️⃣ !!!!!!

**Review by Arqu — APPROVED** [review]
rubber 🔨

**Review by dignifiedquire — COMMENTED** [review]
LFG 🎉

**Pull request merged**